diagnostic.md

```diff
@@ -16,13 +16,13 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+.., cd wdi, cd try.
```

Comment:
Almost, remember that `pwd` prints the whole working directory. Correct path would be `Users/grace/wdi/try`